h/app.py

```diff
@@ -85,15 +85,13 @@ def includeme(config):
     # retrieving services bound to the request.
     config.include("pyramid_services")
 
-    # Configure the transaction manager to support retrying retryable
-    # exceptions, and generate a new transaction manager for each request.
     config.add_settings(
         {
-            "tm.attempts": 3,
```

Comment:
Setting is no longer needed (3 is the default) and using it triggers a deprecation warning